meson-config.txt

```diff
@@ -1,3 +1,8 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+# override toolchain configuration defaults
 [constants]
-march = 'rv32imc'
+march = 'rv32imc' # don't enable bitmanip for now
```

Comment:
Nit: You might not know, but if you do, I would inform the condition to enable bitmanip. 
i.e : "Dont enable bitmanip before version x.xx.x of the toolchain."

Reply:
It's just a policy decision for the project. There's no toolchain impediment. Thanks!